src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -936,13 +936,16 @@ class AccessionStore(
             .and(ACCESSIONS.STATE_ID.`in`(AccessionState.activeValues)))
   }
 
-  fun getSummaryStatistics(projectId: ProjectId): AccessionSummaryStatistics {
+  fun getSummaryStatistics(
+      facilityId: FacilityId,
+      projectId: ProjectId
+  ): AccessionSummaryStatistics {
     requirePermissions { readProject(projectId) }
 
     return getSummaryStatistics(
         DSL.select(ACCESSIONS.ID)
             .from(ACCESSIONS)
-            .where(ACCESSIONS.PROJECT_ID.eq(projectId))
+            .where(ACCESSIONS.PROJECT_ID.eq(projectId).and(ACCESSIONS.FACILITY_ID.eq(facilityId)))
```

Comment:
It'd be more consistent to move the `.and` out one level so it's invoked on the return value of `where()` rather than being part of `where()`'s argument. Or to be more succinct: insert a right parenthesis before `.and` and remove one from the end of the line, which will cause this new `and` to line up with the existing one on the next line.